modules/cluster-logging-elasticsearch-audit.adoc

```diff
@@ -68,7 +68,7 @@ spec:
      secret:
        name: es-audit
    - name: secureforward-offcluster
-     type: "fluentdForward"
+     type: "collectorForward"
```

Comment:
The type is still `fluentdForward`. Probably a greedy search/replace editor mistake here.